### DISCUSSION TITLE
REPLACE or REGEXP_REPLACE  in prisma

### DISCUSSION THREAD

**Discussion by ahmadkam72** [prisma/prisma#22505] [Q&A]
### Question

i want to tool that do what REPLACE or REGEXP_REPLACE do in mysql and postgres,
in prisma.
can any one help?

**Comment by ludralph** [comment]
Hi @ahmadkam72 👋 

Thank you for raising this question.

Prisma currently does not support direct usage of SQL functions like `REPLACE` or `REGEXP_REPLACE` within its query API.

For now, the alternative is to use [`$queryRaw`](https://www.prisma.io/docs/orm/prisma-client/queries/raw-database-access/raw-queries) for complex SQL functions that are not supported by the Prisma Client API. 

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
